<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcordancerWidget</class>
 <widget class="QMainWindow" name="ConcordancerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutMain">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterTB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayoutTop">
        <item row="0" column="0" colspan="2">
         <widget class="QSplitter" name="splitterLR">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QDockWidget" name="dockWidgetMetadataFilters">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string>Search in...</string>
           </property>
           <widget class="QWidget" name="dockWidgetMetadataFiltersContents">
            <layout class="QGridLayout" name="gridLayoutMetadataFilters">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dockWidgetQueryDefinition">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>4</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string>Query Definition (what to search for)</string>
           </property>
           <widget class="QWidget" name="dockWidgetQueryContents">
            <layout class="QGridLayout" name="gridLayoutQueryDefinition">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dockWidgetLevelsAttributes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string>Levels/Attributes Displayed in Results</string>
           </property>
           <widget class="QWidget" name="dockWidgetLevelsAttributesContents">
            <layout class="QGridLayout" name="gridLayoutLevelsAttributes">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCommandLinkButton" name="commandSearchOccurrences">
          <property name="text">
           <string>Search for Occurrences</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QDockWidget" name="dockWidgetResults">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="windowTitle">
        <string>Results</string>
       </property>
       <widget class="QWidget" name="dockWidgetResultsContents">
        <layout class="QGridLayout" name="gridLayoutResults">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
